tests/contrib/hooks/test_gcp_dataflow_template_hook.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is there a reason why this test can't be part of test_gcp_dataflow_hook? Similarly for tests/contrib/operators/test_dataflow_template_operator --> test_dataflow_operator.

Reply:
No reason really, I just kept the DataflowPythonOperator test separated, will merge.